python/base_computer/ship_view.py

```diff
@@ -0,0 +1,304 @@
+import locale
+import math
+import json
+
+
+newline = "#n#"
+red = "#c1:.3:.3#"
+green = "#c0:1:.5#"
+light_grey = "#c.75:.9:1#"
+grey = "#c.6:.7:.8#"
+light_yellow = "#c.675:.925:.825#"
+end_color = "#-c"
+
+non_combat_speed_multiplier = 1
+megajoules_multiplier = 1
+
+llama_begin = {}
+
+with open('config.json', 'r') as file:
+    data = json.load(file)
+    non_combat_speed_multiplier = data['components']['drive']['non_combat_mode_multiplier']
+    megajoules_multiplier = data['constants']['megajoules_multiplier']
```

Comment:
since `config.json` would have to be in a specific place we need to either:
(a) define a working directory for the active "game"
(b) expose these as Python Objects instead of making each script load it if it wants access. This would eliminate the `open` and `json.load` steps; and we'd have to define the interface - either as a pure Python Dictionary/Object or as something else. A first step might be just having a `load` method that would load the contents and return the `data` object like you have above.

I'd rather `b` between the two.

Reply:
Thinking more about this...option `a` might actually be a better solution for game asset data. But let's evolve the conversation on this beyond just this PR.